### ISSUE TITLE
Move "Kubernetes Installation" under "Distributions"

### ISSUE THREAD

**Issue by thesuperzapper** [kubeflow/website#2567]
Currently there is a confusingly named `Kubernetes Installation` section under `Getting Started`, which contains ​information about the "Vanilla Istio-Dex" distribution.

In the interest of not treaing it any differnt to the other distributions, it should be moved under "Distributions / kfctl".

## propsoed structure is 
```
Distributions:
- kftcl
   - Overview [moved from "Getting Started / Kubernetes Installation / Overview of Deployment on an Existing Kubernetes"] Cluster
   - ... existing stuff...

- ...
```

**Comment by thesuperzapper** [comment]
What are your thoughts @RFMVasconcelos @Bobgy?

**Comment by RFMVasconcelos** [comment]
@thesuperzapper I fully agree with this, it is an overview of `kfctl`. 

It includes a compatibility table which we might want to move to a more central place.

**Comment by RFMVasconcelos** [comment]
Solved in #2569
